Doc/howto/functional.rst

```diff
@@ -1166,19 +1152,6 @@ Or the :func:`sum` built-in and a generator expression::
 
 Many uses of :func:`functools.reduce` are clearer when written as ``for`` loops.
 
-Fredrik Lundh once suggested the following set of rules for refactoring uses of
-``lambda``:
-
-1. Write a lambda function.
-2. Write a comment explaining what the heck that lambda does.
-3. Study the comment for a while, and think of a name that captures the essence
-   of the comment.
-4. Convert the lambda to a def statement, using that name.
-5. Remove the comment.
-
-I really like these rules, but you're free to disagree
-about whether this lambda-free style is better.
-
 
 Revision History and Acknowledgements
```

Comment:
Is it necessary to keep this section?

Reply:
I think the history can be removed, Acknowledgements could be moved to the end (after References) or removed. The references section should also probably have the 'General' subhead removed (I can't add in-line comments there).

Reply:
Agreed; we can drop it.